src/index.html

```diff
@@ -4,9 +4,31 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/59ba0ac1be.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
     <title>To DO List</title>
 </head>
 <body>
-    <h1>To do List in order of importance</h1>
+    <div class="wrapper">
+        <header>
+            <h1>Today's To Do</h1> 
+            <i class="fa-solid fa-arrows-rotate fa-beat"></i>
+        </header>
+        <main>
+            <hr>
+            <div class="form">
+                <input type="text" placeholder="Add to your list..." name="todo" class="todo">
+                <i class="fa-solid fa-arrow-right-to-bracket fa-beat"></i>
+            </div>
+            <hr>
+            <ul class="all-activities"></ul>
+            <div class="clear">
+                <p>Clear all completed</p>
+            </div>
+        </main>
+    </div>
+    
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228282854-809cba20-6da0-4819-a8b4-6e47c2866766.png)

- [ ] Kindly remove this title shown in the image above to make your readme more descriptive and professional 👍 First you already have a title after Table of contents and second your project is not Awesome books so because of these two reason it would be great if you remove this title 😊